crypto-sodium/lib/Crypto/Hash.hs

```diff
@@ -0,0 +1,59 @@
+-- SPDX-FileCopyrightText: 2021 Serokell
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+{-# LANGUAGE ExplicitNamespaces, TypeOperators, TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
+
+-- | Hashing
```

Comment:
Please align the Haddock with that of the `NaCl.Hash` module. (Add a full stop at the end, add an example, etc.)